.github/workflows/unittests.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tf-version: [1.14.0, 1.15.2, 2.0.1, 2.1.0]
+        tf-version: [1.14.0, 1.15.2, 2.0.1, 2.1.0, 2.2.0rc4]
```

Comment:
2.2.0rc4 -> 2.2.0?

Reply:
See my comment above:
> Currently the 2.2.0 pip package hasn't been uploaded but 2.2.0rc4 includes all relevant Python changes so it should be fine to use it for testing.

I'd be fine to wait for the 2.2.0 stable pip package, but I wanted to check that CI is still happy with it.